webpack/assets/javascripts/react_app/components/PasswordStrength/index.js

```diff
@@ -0,0 +1,22 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as actions from './PasswordStrengthActions';
+import { doesPasswordsMatch } from './PasswordStrengthSelectors';
+import reducer from './PasswordStrengthReducer';
+
+import PasswordStrength from './PasswordStrength';
+
+// map state to props
+const mapStateToProps = ({ passwordStrength }) => ({
+  doesPasswordsMatch: doesPasswordsMatch(passwordStrength),
+});
+
+// map action dispatchers to props
+const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
+
+// export reducers
+export const reducers = { passwordStrength: reducer };
```

Comment:
The `passwordStrength` key must match the key in `mapStateToProps`, otherwise, it will break!
This is the reason i decided to move the key deceleration from `reducers/index.js` into this object.

I think it would be more reliable if we keep all the configuration of the `PasswordStrength` inside the folder, so the `combineReducers` method should consume the key and the reducer.